topostats/default_config.yaml

```diff
@@ -18,6 +18,13 @@ filter:
     upper: 1.0
   gaussian_size: 1.0121397464510862 # in px
   gaussian_mode: nearest
+remove_scars:
```

Comment:
Discussion really, should this remain as a separate set of options or should it be nested within `filter:`?

If the later then this chunk needs indenting to...

```
  remove_scars:
    run: true
    removal_iterations: 2
    threshold_low: 0.250
    threshold_high: 0.666
    max_scar_width: 4
    min_scar_length: 16
```

Reply:
Nested 👍